### ISSUE TITLE
redownload cover.jpg in better quality

### ISSUE THREAD

**Issue by rudiratlos** [vitiko98/qobuz-dl#260]
How can i re-download better quality of cover.jpg and artwork.
I forgot to give the arguments --embed-art --og-cover
I don't want to use --no-db, it would download all tracks.

**Comment by ralioc** [comment]
Yes, kind of, but only if you use the database save of all your music you've had downloaded. I've worked around this by going to my qobuz download folder. Do a search for .jpg & .png files. Delete them, then re-run your script with --no-db. It will recognize all of your already downloaded files and re-download any artwork missing from those directories. It takes a little longer as qobuz-dl needs to verify all missing files.

Just a quick note... this would be for all the music in the folder. Technically you could remove all the artwork in selected directories and run the script with --no-db after removing the specified images. Either way, it would recognize files that already exist and bypass them, leaving only the artwork that's left.
